frontend/src/component/splash/SplashOverlay/SplashOverlay.tsx

```diff
@@ -41,43 +33,55 @@ export const SplashOverlay = () => {
     const { splash, refetchSplash } = useAuthSplash();
     const { setSplashSeen } = useSplashApi();
 
+    const [closedSplash, setClosedSplash] = useState(false);
+
+    const releaseManagementV3Enabled = useFlag('releaseManagementV3Splash');
+
     const splashId = searchParams.get('splash');
     const isKnownId = splashId ? isKnownSplashId(splashId) : false;
 
     useEffect(() => {
-        if (!user || !splash || splashId) return;
+        if (!user || !splash || splashId || user.isAPI || closedSplash) return;
 
-        if (user.isAPI) return;
-
-        const unseenSplashId = activeSplashIds.find(
-            (id) => !hasSeenSplashId(id, splash),
+        const hasSeenReleaseManagementV3 = Boolean(
+            splash['release-management-v3'],
         );
 
-        if (unseenSplashId) {
-            searchParams.set('splash', unseenSplashId);
```

Comment:
Are we deliberately hardcoding release management? Will previous splash ids still work?

Reply:
Yes, we don't really want to have multiple splash screens active in the system at the same time. Every time we add a splash screen it will be a deliberate decision and for a specific cause.